migration/migration_test.go

```diff
@@ -6,17 +6,20 @@ import (
 	"sync"
 	"testing"
 
-	"github.com/almighty/almighty-core/configuration"
+	configurationHandler "github.com/almighty/almighty-core/configuration"
 	"github.com/almighty/almighty-core/resource"
 	_ "github.com/lib/pq"
 	"github.com/stretchr/testify/assert"
 )
 
+var configuration *configurationHandler.ConfigurationData
+
 func TestConcurrentMigrations(t *testing.T) {
 	resource.Require(t, resource.Database)
 
 	var err error
-	if err = configuration.Setup(""); err != nil {
+	configuration, err = configurationHandler.Setup("")
```

Comment:
wouldn't it be easier to do: config := configuration.Setup(). Then you don't have to do the import alias.